COM-404: Adding french translation for medication duration units

rubailly opened the pull request.

rubailly committed:
COM-404: Adding french translation for medication duration units

lauravignoli commented:
@rubailly Please have a look at some failures. The tests are failing but also please fix this:

https://github.com/jembi/cameroon-openmrs-module-bahmniapps/blob/COM-404/ui/app/clinical/common/models/drugOrderViewModel.js#L399

**Test error:**

Firefox 56.0.0 (Linux 0.0.0) treatmentConfig should initialize duration units FAILED
	Expected [ { name : 'Day(s)/ Jour(s)', factor : 1 }, { name : 'Week(s)/ Semaine(s)', factor : 7 }, { name : 'Month(s)/ Mois', factor : 30 } ] to equal [ { name : 'Day(s)', factor : 1 }, { name : 'Week(s)', factor : 7 }, { name : 'Month(s)', factor : 30 } ].

rubailly committed:
COM-404: Fixing unit tests after adding french translation for medication duration units

rubailly committed:
COM-404: Month(s) should be changed to Month(s)/ Mois since we added french translation

rubailly committed:
COM-404: Month(s) should be changed to Month(s)/ Mois since we added french translation

tmvumbi2 committed:
COM-404 Fix label format

Adds space before slashes; and also uses digit for numbers where the English version uses digit (to maintain a coherent translation)
